user-guide/content/docs/installation.md

```diff
@@ -34,14 +34,28 @@ docker pull inputoutput/cardano-wallet:v2020.4.7-byron
 For version compatibility between components, please refer to compatibility matrix on each component main page (e.g. [cardano-wallet#latest-releases](https://github.com/input-output-hk/cardano-wallet#latest-releases)).
 {{</hint>}}
 
+## Docker compose
```

Comment:
Be careful that the Adrestia user-guide isn't wallet-centric. Any documentation that is project-specific should be in the project repository itself so that we can point to it from the guide. Here, these bits should be part of the docker guide in cardano-wallet's wiki.

Reply:
OK.